### PR TITLE
{Diskpool} Fix issue for generation issue

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Diskpool

File: src/diskpool/azext_diskpool/generated/action.py [hunk 1]
```diff
@@ -57,56 +29,68 @@ def get_action(self, values, option_string):
...
             super(AddDiskPoolCreateDisks, self).__call__(parser, namespace, item, option_string)
 
     def get_action(self, values, option_string=None):
         try:
-            value_chunk_list = [values[x: x + 1] for x in range(0, len(values), 1)]
+            value_chunk_list = [values[x:x + 1] for x in range(0, len(values), 1)]
             value_list = []
             for chunk in value_chunk_list:
-                id = chunk
+                id, = chunk
```
Comment (evelyn-ys) [inline src/diskpool/azext_diskpool/generated/action.py:55]: extra `,`?

**Review by evelyn-ys — APPROVED** [review]

**Comment by jsntcy** [comment]
What issue will this PR fix? @Juliehzl

**Review by jsntcy — APPROVED** [review]

**Comment by Juliehzl** [comment]
> What issue will this PR fix? @Juliehzl

discussed offline

**Pull request merged**